movie.sh

```diff
@@ -24,45 +30,26 @@ do
 done < "$input"
 rm mov.txt
 rm submov.txt
-rate=$(printf %.2f $rating)
+#Convert String to Float
+rate=$(printf %.1f $rating)
 echo $rate
 }
 
-
+rm out.txt
+#Fetching Movies and removing _
 ls movies > movlist.txt
 sed 's/_/./g' movlist.txt > mov2.txt
 input2="mov2.txt"
+echo Rating '     '  Novie >> out.txt
```

Comment:
Typo. And see this as well
https://google.github.io/styleguide/shell.xml#Quoting